cekit/templates/template.jinja

```diff
@@ -1,3 +1,29 @@
+{% macro repo_create(pkg_manager, repositories) -%}
+    {% if pkg_manager in ['microdnf', 'yum'] %}
+        {% for repo in repositories %}
+repos/{{ repo.filename }} \
+        {% endfor %}
+    /etc/yum.repos.d/
+    {% endif %}
+{%- endmacro %}
+{% macro repo_remove(pkg_manager, repositories) -%}
+    {% if pkg_manager in ['microdnf', 'yum'] %}
+rm{% for repo in repositories %} /etc/yum.repos.d/{{ repo.filename }}{% endfor %}
+    {% endif %}
+{%- endmacro %}
+{% macro repo_install(pkg_manager, rpm) -%}
+    {% if pkg_manager in ['microdnf', 'yum'] %}
+{{ pkg_manager }} install -y {{ rpm }} \
+    && {{ pkg_manager }} clean all && rm -rf /var/cache/yum
+    {% endif %}
+{%- endmacro %}
+{% macro pkg_install(pkg_manager, packages) -%}
+    {% if pkg_manager in ['microdnf', 'yum'] %}
+{{ pkg_manager }} install -y {%- for package in packages %} {{ package }}{% endfor %} \
```

Comment:
I was thinking that you would have a separate macro for each package manager, e.g. `dnf_install` and `yum_install`.  That said, the rest of the changes are above and beyond.

Reply:
Why adding multiple macros when you can have one for all? I think it'd be easier to extend
I decided to add macros for:
* create/install/delete a repo
* install a package

Reply:
Indeed, support for e.g. Debian is more than just changing to apt; the package names will be different, the paths will be different…

Reply:
The idea here (at least "mine") is to allow cekit to work with other images than centos/rhel. In case you use Debian, your image.yml will define different packages to install or else you will build it with an overrides file.